test/system/005-info.bats

```diff
@@ -194,6 +194,13 @@ host.slirp4netns.executable | $expr_path
 
     ln -s /home $new_home
 
+    # Remove volume directory. This doesn't break Podman but can cause our DB
+    # validation to break if Podman misbehaves. Ref:
+    # https://github.com/containers/podman/issues/23515
+    # (Unfortunately, we can't just use a new directory, that will just trip DB
+    # validation that it doesn't match the path we were using before)
+    rm -rf $PODMAN_TMPDIR/$HOME/.local/share/containers/storage/volumes
+
```

Comment:
Did you actually test this with an older podman binary? This test always passes without your patch for me locally so this is a NOP I think.